lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -211,8 +237,9 @@ def va_profile_opt_in_out_lambda_handler(  # noqa: C901
         assert integration_testing_public_cert is not None
 
     # Authenticate the POST request by verifying the JWT signature.
+    auth_header_value = headers.get('Authorization', headers.get('authorization', ''))
```

Comment:
I don't think a var needs to be declared like this, it's only used once?

Reply:
Updated.